### ISSUE TITLE
Separate Accessories tile

### ISSUE THREAD

**Issue by Anraza-Valtieri** [torifat/xiaomi-mi-air-purifier#11]
In Homekit, some accessories are able to be separated into multiple tiles instead of bundling into one single tile. Is it possible to get a feature like that for this?

**Comment by ifeign** [comment]
Second this, the combined accessory makes it very difficult to view more detailed info, you have to go into the settings to get to the list of combined accessories, then press again on the air purifier accessory to see things like auto/manual, filter life, child lock etc. Older plugins show this on the main screen.

Additionally, we need an additional switch accessory for sleep mode.

**Comment by torifat** [comment]
> Second this, the combined accessory makes it very difficult to view more detailed info, you have to go into the settings to get to the list of combined accessories, then press again on the air purifier accessory to see things like auto/manual, filter life, child lock etc. Older plugins show this on the main screen.

I can probably consider adding an option for it but personally I find it really noisy.

I can add an option to make the AirPurifier optional so that you can multiple seperate tiles like this:
```json
{
  "platforms": [
    {
      "platform": "XiaomiMiAirPurifier",
      "config": {
          "address": "<YOUR_DEVICE_IP_ADDRESS>",
          "token": "<YOUR_DEVICE_TOKEN>"
      },
      "accessories": [
        {
          "name": "Air Purifier"
        },
        {
          "name": "Air Quality",
          "airPurifier": false,
          "enableAirQuality": true
        },
        {
          "name": "Temperature",
          "airPurifier": false,
          "enableTemperature": true
        },
      ]
    }
  ]
}
```

> Additionally, we need a switch accessory for sleep mode.

What do you mean by sleep mode? Isn't power off is same as sleep mode?

**Comment by ifeign** [comment]
By sleep mode, I meant quiet mode. Sorry about that.

**Comment by torifat** [comment]
> By sleep mode, I meant quiet mode. Sorry about that.

I was looking for a nicer way to add all the modes -> `auto`, `sleep`, `favorite`, `custom`. I don't like switches for them 😞  But, it seems there's no other way. I will dig a bit more and then add them.

**Comment by ifeign** [comment]
Could you make 0 fan speed = quiet mode?

**Comment by torifat** [comment]
> Could you make 0 fan speed = quiet mode?

I can but it will be confusing. You won't know from the UI if it's in quite mode or not.

**Comment by ifeign** [comment]
True. I guess a switch is best. In the plugin that I used to use it just
has a switch that I give change to a fan in the home app. Could this switch
just be part of the combined accessory?

For reference:
https://github.com/YinHangCode/homebridge-mi-airpurifier

**Comment by ifeign** [comment]
Any updates on this? Your plugin is so close to being the best Mi Purifier plugin, if you could add a Quiet Mode switch and allow people to show separate tiles for the accessory, this would pretty much be feature complete.
